model/wire.cc

```diff
@@ -543,6 +537,26 @@ namespace
     c.insert(c.begin()+n,h.begin(), h.end());
     coords(c);
   }
+
+  // For ticket 1092. Reinstate delete handle user interaction
+  void Wire::deleteHandle(float x, float y)
+  {
+    auto c=coords();
+    unsigned n=0; // nearest index
+    float closestD=d2(c[0],c[1],x,y);
+    for (size_t i=2; i<c.size()-1; i+=2)
+      {
+        float d=d2(c[i],c[i+1],x,y);
+        if (d<closestD)
+          {
+            closestD=d;
+            n=i;
+          }
+      }	  
+    assert(n<c.size()-1);
+    c.erase(c.begin()+n);
```

Comment:
Don't you have to erase the y coordinate as well? c.erase(c.begin()+n, c.erase(c.begin()+n+2) should erase both the x and y coordinate values.

Reply:
Thanks, I've made it c.erase(c.begin()+n, c.erase(c.begin()+n+1));

With n+2 it deletes more than one handle on double-click.

Reply:
Sorry I meant to say c.erase(c.begin()+n, c.begin()+n+2), which erases two items starting at c.begin()+n in one hit. What you ended up with is pretty much equivalent, although terribly obscure, and with two calls to erase, so the trailing data will be copied twice. I'll approve this, and fix it in master branch.